libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -270,6 +268,14 @@ public void finishNavigationView() {
    * @param options with containing route / coordinate data
    */
   public void startNavigation(NavigationViewOptions options) {
+    // Initialize navigation with options from NavigationViewOptions
+    navigationViewModel.initializeNavigationOptions(getContext().getApplicationContext(),
+      options.navigationOptions().toBuilder().isFromNavigationUi(true).build());
```

Comment:
Is it possible for `options.navigationOptions()` to be null?

Reply:
The `NavigationViewOptions` `builder()` will always provide `MapboxNavigationOptions` with the defaults - so someone would have to explicitly set it to null

Reply:
Annotate the parameter with `@NonNull` then